zkevm-circuits/src/evm_circuit/execution/end_tx.rs

```diff
@@ -203,6 +250,26 @@ impl<F: Field> ExecutionGadget<F> for EndTxGadget<F> {
             coinbase_balance,
         )?;
 
+        let current_cumulative_gas_used: u64 = if tx.id == 1 {
+            0
+        } else {
+            let rw = &block.rws[(
```

Comment:
I think this can be simplified if the `cumulative_gas_used` field is stored in the `tx: &Transaction`. See comment above.